docs/modules/operation/pages/deep-dive/grpc-exporter/grpc-exporter.adoc

```diff
@@ -17,13 +17,15 @@ admin@opennms()> config:property-set bsm.host bsm.onmshs.local:1443 <1>
 admin@opennms()> config:property-set spog.host nms-inventory.onmshs.local:1443 <2>
 admin@opennms()> config:property-set tenant.id opennms-prime <3>
 admin@opennms()> config:property-set tls.cert.path /opt/opennms/etc/tls.cert <4>
+admin@opennms()> config:property-set zenithConnect.enabled true <5>
```

Comment:
It still doesn't follow convention that was used here for all other config. lowercase for all.

zenith.connect.enabled.

Does this follow under optional config ?  see below.